src/vs/workbench/contrib/welcome/page/browser/welcomePage.ts

```diff
@@ -264,14 +266,17 @@ class WelcomePage extends Disposable {
 		@IExtensionTipsService private readonly tipsService: IExtensionTipsService,
 		@IExtensionsWorkbenchService private readonly extensionsWorkbenchService: IExtensionsWorkbenchService,
 		@ILifecycleService lifecycleService: ILifecycleService,
-		@ITelemetryService private readonly telemetryService: ITelemetryService
+		@ITelemetryService private readonly telemetryService: ITelemetryService,
+		// {{SQL CARBON EDIT}}
```

Comment:
We should put the carbon edit comment inline with the edit. It makes merge conflicts less likely and makes it easier to see the edits when searching for them